### ISSUE TITLE
Move Axe Details closer to Accessibility

### ISSUE THREAD

**Issue by mgifford** [mgifford/daily-dap#54]
Axe details should be right next to the Accessibility in the table. 

https://mgifford.github.io/daily-dap/docs/reports/daily/2026-03-16/index.html#top-urls-heading

It is currently: 
URL | Traffic | Scan status | CWV | Performance | Accessibility | Best Practices | SEO | Total findings | Critical/Serious | Failure reason | Axe details
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

I want to have the order more like:

URL | Traffic | Scan status | CWV | Performance | Accessibility | Axe details | Best Practices | SEO | Total findings | Critical/Serious | Failure reason
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
